be/src/exec/pipeline/chunk_source.h

```diff
@@ -11,6 +11,12 @@
 
 namespace starrocks {
 class RuntimeState;
+
+namespace workgroup {
+class WorkGroup;
+using WorkGroupPtr = std::shared_ptr<WorkGroup>;
+} // namespace workgroup
+
```

Comment:
There is a new header file `exec/workgroup/work_group_fwd.h` contains these forward declarations, so maybe we could just include this file.